app/config/dev-config-n0npax.yaml

```diff
@@ -0,0 +1,34 @@
+# dev config
```

Comment:
we don't want to store developer-specific configuration in the repo, please remove this

Reply:
sorry, I thought I removed this file, my bad